adafruit_mpu6050.py

```diff
@@ -74,13 +75,36 @@
 STANDARD_GRAVITY = 9.80665
 
 
+class CLockSource:  # pylint: disable=too-few-public-methods
```

Comment:
I think the `L` here is accidentally capitalized.

Reply:
Wow.   Copy and Pasts got me there.  Thank you!